src/viur_cli/local.py

```diff
@@ -7,7 +10,28 @@
 @click.argument("name", default='develop')
 @click.argument("additional_args", nargs=-1)
 def run(name, additional_args):
-    """start your application locally"""
+    """
+        Start your application locally.
+
+        The 'run' command allows you to start your ViUR application locally. You can specify the configuration to use
+        and provide additional arguments to the 'app_server' command.
+
+        :param name: str, default: 'develop'
+            The name of the configuration to use. The 'develop' configuration is the default.
+
+        :param additional_args: List[str]
+            Additional arguments to pass to the 'app_server' command when running the application.
+
+        Example Usage:
+        ```shell
+        viur run develop --port 8080
+        viur run production
+        ```
+
+        The 'run' command launches your ViUR application locally specified configuration and optional arguments.
+
+        :return: None
+    """
```

Comment:
```suggestion
    """Start your application locally.

    The 'run' command allows you to start your ViUR application locally. You can specify the configuration to use
    and provide additional arguments to the 'app_server' command.

    :param name: str, default: 'develop'
        The name of the configuration to use. The 'develop' configuration is the default.

    :param additional_args: List[str]
        Additional arguments to pass to the 'app_server' command when running the application.

    Example Usage:
    ```shell
    viur run develop --port 8080
    viur run production
    ```

    The 'run' command launches your ViUR application locally specified configuration and optional arguments.

    :return: None
    """
```